CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@
    1. [Minimal Tool Requirements](#minimal-tool-requirements)
    2. [Getting Started](#getting-started)
    3. [Building the Docker Image](#building-the-docker-image)
-   4. [Building the APK locally](#building-the-apk-locally)
+   4. [Building the APK](#building-the-apk)
```

Comment:
A general comment for this file:
Please clean this file up:
1. At some places you call the "release" version "stable" version. Unify this.
2. At the beginning you say that you will need docker but somewhere below, you have a heading like "Without docker environment".
3. In the release section, you only talk about FDroid. Add information for the playstore

Reply:
You are right. However, I would like to do this in a separate PR as stated above.